public/500.html

```diff
@@ -67,55 +78,65 @@
+        <p class="text-muted">&copy; 2017 The Broad Institute of MIT & Harvard</p>
```

Comment:
Correct year and [preferred form](https://drive.google.com/file/d/0B877YjdN2Vv9ZkdPNElpSTZmTDg/view) (see page 7) for institute name :

```suggestion
        <p class="text-muted">&copy; 2020 Broad Institute of MIT and Harvard</p>
```